desktop/src/com/unciv/app/desktop/PlatformSpecificHelpersDesktop.kt

```diff
@@ -2,20 +2,11 @@ package com.unciv.app.desktop
 
 import com.badlogic.gdx.backends.lwjgl3.Lwjgl3ApplicationConfiguration
 import com.unciv.ui.utils.GeneralPlatformSpecificHelpers
-import java.net.InetAddress
 
 class PlatformSpecificHelpersDesktop(config: Lwjgl3ApplicationConfiguration) : GeneralPlatformSpecificHelpers {
     val turnNotifier = MultiplayerTurnNotifierDesktop()
     init {
-        config.setWindowListener(turnNotifier);
-    }
-
-    override fun isInternetConnected(): Boolean {
-        return try {
-            InetAddress.getByName("8.8.8.8").isReachable(500)  // Parameter timeout in milliseconds
```

Comment:
Sigh could it be that this is the reason why my checks always failed on Pi?
Proxy increases latency by a good margin and 500 is really not enough.
It is not enough even without proxy. And will fail 5-10% of times.
Who even sets timeout to only 0.5 seconds, I am astonished that nobody pointed this out.